po/es.po

```diff
@@ -311,9 +309,7 @@ msgid "Pd window"
 msgstr "Ventana de Pd"
 
 msgid "Match whole word only"
-msgstr ""
-"Comparar solo con\n"
```

Comment:
this `\n` was intentional as it fits better in the button.

I'll try to reintroduce it via `weblate.org`

Reply:
i thought so, but didn't really check.

maybe you can add a note on weblate

Reply:
done